package.json

```diff
@@ -7,8 +7,9 @@
     "@typespec/openapi": "^0.49.0-dev.4",
     "@typespec/openapi3": "^0.49.0-dev.10",
     "@typespec/rest": "^0.49.0-dev.3",
-    "@typespec/http": "^0.49.0-dev.0",
-    "@typespec/versioning": "^0.49.0-dev.0"
+    "@typespec/http": "0.49.0-dev.0",
```

Comment:
the latest version 0.49.0-dev.3 @typespec/http library seems to have some issues, it cause the compilation fail even with the OAI3 emitter.